### PR TITLE
Release preparation

### PR DESCRIPTION
**Pull request by WieFel**
After this is merged, i publish first version to pub.dev!

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: tinycolor2
 description: Flutter Color manipulation and conversion, ported from JS tinycolor2
 version: 2.0.0
-homepage: https://github.com/TinyCommunity/tinycolor2
+homepage: https://tinycommunity.dev/
```
Comment (calvintam236) [inline pubspec.yaml:4]: I think this refers to repo url?

Comment (WieFel) [reply pubspec.yaml:4]: https://dart.dev/tools/pub/pubspec#homepage
https://dart.dev/tools/pub/pubspec#repository
We could add `repository` as well and keep homepage as it is now?

Comment (calvintam236) [reply pubspec.yaml:4]: Yes, also add https://dart.dev/tools/pub/pubspec#issue-tracker

**Comment by calvintam236** [comment]
@WieFel I think we need to merge #30 to run some tests on here before releasing it.

File: pubspec.yaml (hunk 1 above)
Comment (WieFel) [reply pubspec.yaml:4]: Well ok, i can also revert the commit. It is also ok to put repo page as `homepage`...

**Comment by WieFel** [comment]
> @WieFel I think we need to merge #30 to run some tests on here before releasing it.

But for getting the credentials we need to first publish it once manually

**Comment by calvintam236** [comment]
> > @WieFel I think we need to merge #30 to run some tests on here before releasing it.
> 
> But for getting the credentials we need to first publish it once manually

That’s `cd` task. We need to run `ci` here. `cd` not working at this moment is fine.

**Comment by WieFel** [comment]
@calvintam236 well ok. Let's merge it then

**Comment by WieFel** [comment]
@calvintam236 should we remove these lines?
https://github.com/TinyCommunity/tinycolor2/blob/b605013805cdd02e343c64d9407f5aaf1f0a3773/.github/workflows/ci.yml#L4-L7

I think it is enough to run CI on PR. Currently, the actions run twice per push, once for push + once for PR...

**Comment by calvintam236** [comment]
> @calvintam236 should we remove these lines?
> https://github.com/TinyCommunity/tinycolor2/blob/b605013805cdd02e343c64d9407f5aaf1f0a3773/.github/workflows/ci.yml#L4-L7
> 
> I think it is enough to run CI on PR. Currently, the action runs twice per push, once for push + once for PR...

It is set to trigger when PR is ready to review or push. If someone else make a PR from their fork, only PR one will be triggered. Since we are not paying for the GH action, please let it run on every push.

**Comment by calvintam236** [comment]
@WieFel is it all formatted inside example/?

**Comment by WieFel** [comment]
> @WieFel is it all formatted inside example/?

Yes, I suppose CI format check would also trigger if something is wrong there!?
Just double checked, and it's ok!

**Comment by WieFel** [comment]
> > @calvintam236 should we remove these lines?
> > https://github.com/TinyCommunity/tinycolor2/blob/b605013805cdd02e343c64d9407f5aaf1f0a3773/.github/workflows/ci.yml#L4-L7
> > 
> > I think it is enough to run CI on PR. Currently, the action runs twice per push, once for push + once for PR...
> 
> It is set to trigger when PR is ready to review or push. If someone else make a PR from their fork, only PR one will be triggered. Since we are not paying for the GH action, please let it run on every push.

Seems that `ready_for_review` is triggered all the time, as long as PR is not set to "draft".
I think more adapt would be to use `review_requested `, so it would run once a review is requested.

**Comment by calvintam236** [comment]
> > > @calvintam236 should we remove these lines?
> > > https://github.com/TinyCommunity/tinycolor2/blob/b605013805cdd02e343c64d9407f5aaf1f0a3773/.github/workflows/ci.yml#L4-L7
> > > 
> > > I think it is enough to run CI on PR. Currently, the action runs twice per push, once for push + once for PR...
> > 
> > 
> > It is set to trigger when PR is ready to review or push. If someone else make a PR from their fork, only PR one will be triggered. Since we are not paying for the GH action, please let it run on every push.
> 
> Seems that `ready_for_review` is triggered all the time, as long as PR is not set to "draft".
> I think more adapt would be to use `review_requested`, so it would run once a review is requested.

we can fix that after merging this. that does not matter for release

**Review by calvintam236 — APPROVED** [review]

**Comment by WieFel** [comment]
Just did a publish dry run:
```
Package validation found the following potential issues:
* Please consider renaming LICENSE.md to `LICENSE`. See https://dart.dev/tools/pub/publishing#important-files.
* Packages with an SDK constraint on a pre-release of the Dart SDK should themselves be published as a pre-release version. If this package needs Dart version 2.12.0-0, consider publishing the package as a pre-release instead.
  See https://dart.dev/tools/pub/publishing#publishing-prereleases For more information on pre-releases.
* The Flutter constraint should not have an upper bound.
  In your pubspec.yaml the constraint is currently `>=1.12.0 <3.0.0`.
  
  You can replace that with just the lower bound: `>=1.12.0`.
  
  See https://dart.dev/go/flutter-upper-bound-deprecation

Package has 3 warnings.
```

Need to fix these things...

**Comment by WieFel** [comment]
Package has 0 warnings.

**Review by calvintam236 — COMMENTED** [review]
Can we make a soft link LICENSE linking back to LICENSE.md since changes made to file not ending in `.md` will trigger GitHub actions

**Comment by WieFel** [comment]
LICENSE will never be changed again

**Review by calvintam236 — APPROVED** [review]

**Pull request merged**